docs/index.rst

```diff
@@ -61,3 +61,30 @@ Features
     how-to
     how-it-works
     changelog
```

Comment:
The `796.doc` file is being listed as binary, and missing the `.rst` extension, can you please take a look?

Reply:
Yes i folowed example of 791.doc that is a word file so i did mine as well.... i think this realy needs  contributing section  But i will change it to rst file  definetly

Reply:
Ahh I see the confusion... actually `791.doc` is a text file:

https://github.com/pytest-dev/pytest-xdist/blob/master/changelog/791.doc

But we missed that during merge, as usually we recommend users to use `.rst` as extension, so that should be `791.doc.rst`.

To be clear, your file should be `796.doc.rst`. 😁 

That is because how [towncrier](https://pypi.org/project/towncrier/) works, it uses the extension to categorize the changelog entries, so it supports `123.feature`, `456.doc`, and so on. It later added support to have an extra extension that defines the file format (`.md` or `.rst`) to avoid confusion, so we use `123.feature.rst`, `456.doc.rst`, and so on.

> i think this realy needs contributing section

I agree. 👍

Reply:
well i actualy open that 791.doc i in libre office so yes i was bamboozeled